core/src/test/java/org/pentaho/platform/util/EmailerTest.java

```diff
@@ -82,4 +108,57 @@ public void setBcc_ReplaceToCommaTest() {
     emailer.setBcc( "bcc@domain.com; another_bcc@domain.com" );
     assertEquals( "bcc@domain.com, another_bcc@domain.com", emailer.getProperties().getProperty( "bcc" ) );
   }
+
+  @Test
```

Comment:
Good job moving the test from scheduler/…/email/EmailerTest.java to core/…/util/EmailerTest.java